lxd/storage_pools.go

```diff
@@ -325,6 +327,16 @@ func storagePoolsPost(d *Daemon, r *http.Request) response.Response {
 		return response.InternalError(err)
 	}
 
+	projectName := projectParam(r)
+	requestor := request.CreateRequestor(r)
+
+	ctx := log.Ctx{}
+	if targetNode != "" {
+		ctx["target"] = targetNode
+	}
+
+	d.State().Events.SendLifecycle(projectName, lifecycle.StoragePoolCreated.Event(req.Name, projectName, requestor, ctx))
```

Comment:
This is good but there's another branch above which bypasses the SendLifecycle (the one that's hit when you finalize the creation of the storage pool in a cluster).